tests/test_variables.py

```diff
@@ -74,6 +76,38 @@ def test_pex_vars_set():
   assert v._get_int('HELLO') is None
 
 
+def test_pex_get_kv():
+  v = Variables(environ={})
+  assert v._get_kv('HELLO') is None
+  assert v._get_kv('=42') is None
+  assert v._get_kv('TOO=MANY=COOKS') is None
+  assert v._get_kv('THIS=WORKS') == ['THIS', 'WORKS']
+
+
+def test_pex_from_rc():
+  with tempfile.NamedTemporaryFile(mode='w') as pexrc:
+    pexrc.write('HELLO=42')
+    pexrc.flush()
+    v = Variables(rc=pexrc.name)
+    assert v._get_int('HELLO') == 42
+
+
+def test_pexrc_precedence():
+  with tempfile.NamedTemporaryFile(mode='w') as pexrc:
+    pexrc.write('HELLO=FORTYTWO')
+    pexrc.flush()
+    v = Variables(environ={'HELLO': 42}, rc=pexrc.name)
+    assert v._get_int('HELLO') == 42
+
+
+def test_rc_ignore():
+  with tempfile.NamedTemporaryFile(mode='w') as pexrc:
+    pexrc.write('HELLO=FORTYTWO')
+    pexrc.flush()
+    v = Variables(environ={'PEX_IGNORE_RC_FILES': True, 'HELLO': 42}, rc=pexrc.name)
+    assert v._get_int('HELLO') == 42
```

Comment:
Actually - this test is doubly broken atm.  The IGNORE env var name is typoed and the value is invalid - must be a string.  Its an accident ints work as `environ=` values, the intent is int([str]) in the parse routine and it happens that int(int) works just fine.

So this works:
```diff
$ git diff -U1
diff --git a/tests/test_variables.py b/tests/test_variables.py
index 124ec6d..447efb5 100644
--- a/tests/test_variables.py
+++ b/tests/test_variables.py
@@ -106,4 +106,4 @@ def test_rc_ignore():
     pexrc.flush()
-    v = Variables(environ={'PEX_IGNORE_RC_FILES': True, 'HELLO': 42}, rc=pexrc.name)
-    assert v._get_int('HELLO') == 42
+    v = Variables(environ={'PEX_IGNORE_RCFILES': 'True'}, rc=pexrc.name)
+    assert 'HELLO' not in v._environ
 ```

Since the fixed up test works - ie the prod code works - I'll merge this in and follow up with a fix to make the test valid.